mirror-wordpress-plugins

```diff
@@ -206,6 +206,10 @@ class PluginUpdater
       puts "==> Skipping download of .zip file since NO_DOWNLOAD_MIRROR variable is set"
     else
       puts "==> Downloading #{@wordpress_download_link}..."
+      response_code = `curl -o /dev/null -s -w "%{http_code}" #{@wordpress_download_link}`.chomp
```

Comment:
Could we add an `-I` to the curl options here to get just the head info? A quick local test using `time` suggests that could save us ~0.5 seconds per request.